### PR TITLE
Update rancher-2.9-support.md

### PR DESCRIPTION
**Pull request by owanturist**
### Summary
Fixes typo in the desired `@rancher/shell` version from 2.9.0 to 2.0.1

### Checklist
- [ ] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [ ] The PR has a Milestone 
- [ ] The PR template has been filled out
- [ ] The PR has been self reviewed 
- [ ] The PR has a reviewer assigned
- [ ] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [ ] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

File: docusaurus/docs/extensions/rancher-2.9-support.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ For more information about the annotation we allow for, check the documentation
 - After the above steps, just publish a new version of the extension. That published version should now be compliant with Rancher 2.9.0.
 
 
-To find out more about the support matrix for Shell versions in regards to Rancher versions, check the support matrix [here](./support-matrix#shell-support-matrix).
\ No newline at end of file
+To find out more about the support matrix for Shell versions in regards to Rancher versions, check the support matrix [here](./support-matrix#shell-support-matrix).
```
Comment (owanturist) [inline docusaurus/docs/extensions/rancher-2.9-support.md:56]: don't see why this is highlighted as an updated 🤔 EOF?

**Review by jordojordo — APPROVED** [review]
lgtm - thanks for the fix!

File: docusaurus/docs/extensions/rancher-2.9-support.md (hunk 1 above)
Comment (aalves08) [reply docusaurus/docs/extensions/rancher-2.9-support.md:56]: probably.... 😛  don't worry

**Review by aalves08 — APPROVED** [review]
LGTM! Thanks for the changes @owanturist 🙇

**Pull request merged**